<!doctype html>
<html lang="en-US">
<head>
	<meta charset="UTF-8" />
	<title>Cerberus X Documentation</title>
	<link rel="stylesheet" type="text/css" href="pagestyle.css" />
</head>
<body>
	<div class="page">
		<header>
			<h1>
				<a href="Home.html"><img src="data/cerberusx.png"/></a>Cerberus X Documentation
			</h1>

		</header>

		<nav class="main">
			<a href="Home.html">Home</a>
			<a href="API Reference.html">API Reference</a>
			<a href="Programming_Language reference.html">Language Reference</a>
			<a href="Modules.html">Modules</a>
			<a href="Indexes.html">Indexes</a>
		</nav>
		
		<nav class="location">
			<a href='Home.html'>Home</a>

			<a href='Programming.html'>Programming</a>

			<a href='Programming_Keywords.html'>Keywords</a>

			<span>Catch</span>

		</nav>

		<main>

<h1>Keyword Catch</h1>
<p>
The Catch keyword is part of the Try/Catch exception-handling construct. See Try for more details on exception handling.
</p>
<h2>Syntax</h2>
<p>
Catch <b>exception</b>
</p>
<h2>Description</h2>
<p>
In the event of an exception being thrown within a Try/Catch block, program flow jumps to the Catch section matching the exception type; after the relevant error-handling code is executed, program flow exits the Try/Catch block.
</p>
<p>
Multiple Catch sections may be declared within a Try/Catch block in order to handle different types of exception. See Try for examples and further details.
</p>
<h2>See also</h2>
<p>
<a href="Programming_Keywords_Try.html">Try</a> | <a href="Programming_Keywords_Throw.html">Throw</a> | <a href="Modules_cerberus.lang_Throwable.html">Throwable</a> | <a href="Programming_Keywords_End.html">End</a><br>
<a href="Programming_Language reference.html">Language reference</a>
</p>

		</main>
	</div>
</body>
</html>